keyboards/converter/adb_usb/adb_usb.h

```diff
@@ -106,4 +106,46 @@ Ported to QMK by Peter Roe <pete@13bit.me>
   { KC_NO, KC_NO, KC_NO, K7B,   KC_NO, KC_NO, KC_NO, K7F   }  \
 }
 
+/* M0118 Apple Standard Keyboard ISO
+ *                     +-------+
+ *                     | power |
+ *                     +-------+
+ * +---+---+---+---+---+---+---+---+---+---+---+---+---+-----+ +---+---+---+---+
+ * |esc| 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 0 | - | = | bks | |clr| = | / | * |
+ * +---------------------------------------------------------+ +---+---+---+---+
+ * | tab | q | w | e | r | t | y | u | i | o | p | [ | ] |Ret| | 7 | 8 | 9 | + |
+ * +-----------------------------------------------------`   | +---+---+---+---+
+ * | caps | a | s | d | f | g | h | j | k | l | ; | ' | # |  | | 4 | 5 | 6 | - |
+ * +---------------------------------------------------------+ +---+---+---+---+
+ * |shif| \ | z | x | c | v | b | n | m | , | . | / |Shif| up| | 1 | 2 | 3 |   |
+ * +---------------------------------------------------------+ +-------+---|ent|
+ * |ctrl|opt  |comnd|                     |comnd |lef|rig|dwn| |   0   | . |   |
+ * +---------------------------------------------------------+ +-------+---+---+
+ */
+#define LAYOUT_m0118_iso( \
+                           K7F,                                              \
+   K35,K12,K13,K14,K15,K17,K16,K1A,K1C,K19,K1D,K1B,K18,K33, K47,K51,K4B,K43, \
+   K30,K0C,K0D,K0E,K0F,K11,K10,K20,K22,K1F,K23,K21,K1E,K24, K59,K5B,K5C,K45, \
+   K39,K00,K01,K02,K03,K05,K04,K26,K28,K25,K29,K27,K2A,     K56,K57,K58,K4E, \
+   K38,K32,K06,K07,K08,K09,K0B,K2D,K2E,K2B,K2F,K2C,K7B,K3E, K53,K54,K55,     \
```

Comment:
The ISO backslash actually is key code `K0A`. Here we just get the `K32` because the swapping key codes function for ISO keyboards are not yet supported by the ADB implementation.
I opened #11168 as a first step towards a solution for this issue. As this is a core change it will not be in master until end of February. Until then having the K32 here does the job.